src/types.ts

```diff
@@ -257,6 +257,18 @@ export type ConfirmStartEvent = ConfirmEvent<ConfirmStartEventPayload>;
 export type ConfirmTapEvent = ConfirmEvent<ConfirmTapEventPayload>;
 
 export type SpeechRecognitionStartEvent = {
-  service: string;
-  serviceOptions: any;
+  service: "unified" | "unified:google" | "unified:aws" | "unified:deepgram";
+  sampleRate?: number;
+  languageCode?: string;
+  encoding?: string;
+  customServiceParameters?: object;
```

Comment:
the `object` type also includes arrays, let's be more specific in what we expect

```suggestion
  customServiceParameters?: Record<string, unknown>;
```